en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+The contents of the report are as follows:
+
+| **Item** | **Description** |
+|-------------------------|--------------------------------------------------------------------------------------------|
+| **Generated Time**      | The date and the time the report was generated.                                            |
+| **Organization**        | The organization to which the report applies. Each report always applies to a specific organization. |
+| **Environment**         | The environment to which the report applies. Each report always applies to a specific environment.  |
+| **Tenant**              | The tenant to which the report applies. Each report always applies to a specific tenant.            |
+| **Reporting Period**    | The time interval for which the report is generated.                                       |
+| **Total request count** | The total request count received by all selected APIs during the report time interval.     |
+
+The table in the report contains the following rows:
+
+| **Item** | **Description** |
+|-----------------------|------------------------------------------------------------------------------|
+| **API Name**          | The name of the API.                                                         |
+| **Application name:** | The name of the application. The name of the subscriber is also provided within brackets. |
+| **Usage Count**       | The request count.                                                           |
+
+#### Download pre-generated reports
+
+These widgets allow you to download monthly reports for the last three months.
+
+![Pre-generated reports](../assets/img/insights/configured-report.png){.cInlineImage-full}
+
+To download a pre-generated report, click **Download** for the relevant month and then click on the required format.
+
+### Alert Configuration
+
+This page allows you to configure alerts. These alerts allow you to proactively monitor your API ecosystem and take corrective measures for any abnormalities you find.
+
+You can configure alerts for each environment within your organization separately. You can configure alerts per API as well as modify/remove already-added alert configurations. Optionally, you can specify a list of emails for each alert configuration.
+
+Alerts are subject to a suppression policy to ensure you do not receive duplicate alert notifications. For each alert you configure, there is a 10-minute suppression window applied by default. The suppression interval is not configurable at present. In addition, note that the suppression policy is applicable per alert configuration.
+
+!!! info
+    - The maximum number of alerts that can be configured is 20 for each organization, environment, and tenant combination.
+    - Adding an email to an alert configuration is optional. The maximum number of emails per alert configuration is limited to 5.
+    
+The alert can be a latency alert or a traffic alert.
+
+#### Latency alerts
```

Comment:
No screenshots for this section?

Reply:
We minimise the use of screenshots in Choreo documentation. In this instance, the screenshot of the page for each step with the relevant field outlined is unnecessary because the mentioned field can be easily located in the UI.

However, used screenshots for the widgets in the Analytics portal so that users can see what it is like when the widgets are populated even when they don't have statistics in their own deployment to try it out.

Reply:
Ack